config/exporter.yaml

```diff
@@ -0,0 +1,3 @@
+# Operator 4
+# Public key: LS0tLS1CRUdJTiBSU0EgUFVCTElDIEtFWS0tLS0tCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBdi82RytFUEhOK2NycGZ1Z01aYTYKQkpPd3ZycjdTNGkvbWprbTU4czN3RXhkVnFQUjZFbXBUa29QQ0tWamlWSTMvUzIvRjdRdm1FNlA1M3N3cEZoOQpCTDZnaHRpRlZtUzBXQVpOejJJODJRT01HdzNxSEk4NmJSS2RuUjdmU2ZjYlFtSlFuRDNyK0xVRngyQngxZENVCndwSTltSDhXNEhrVHE0VkhGTVY5SlRyaWNhYS9SZ0lqd3RjZ25QUkQ3VTBpeU1zeStyS2RpK2xWcHJqNHFqYmwKcEZ6MExGcXVrYWc4cWdxVnpqRVZsVEZSa3VuaWhvNm96TlZhZU1sSnlXRk4rZ1d5YVp5T0xQMjVhNjl2V0hwbQoxeG1nclZuZW9ncEdnc0hNMGZuTHJJRXlTUkNQdjU5UEVwYTZIdGtlODh1ZjkwRFBDa2RaV0xnTXRzdnkwREJDCnJRSURBUUFCCi0tLS0tRU5EIFJTQSBQVUJMSUMgS0VZLS0tLS0K
+OperatorPrivateKey: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBdi82RytFUEhOK2NycGZ1Z01aYTZCSk93dnJyN1M0aS9tamttNThzM3dFeGRWcVBSCjZFbXBUa29QQ0tWamlWSTMvUzIvRjdRdm1FNlA1M3N3cEZoOUJMNmdodGlGVm1TMFdBWk56Mkk4MlFPTUd3M3EKSEk4NmJSS2RuUjdmU2ZjYlFtSlFuRDNyK0xVRngyQngxZENVd3BJOW1IOFc0SGtUcTRWSEZNVjlKVHJpY2FhLwpSZ0lqd3RjZ25QUkQ3VTBpeU1zeStyS2RpK2xWcHJqNHFqYmxwRnowTEZxdWthZzhxZ3FWempFVmxURlJrdW5pCmhvNm96TlZhZU1sSnlXRk4rZ1d5YVp5T0xQMjVhNjl2V0hwbTF4bWdyVm5lb2dwR2dzSE0wZm5McklFeVNSQ1AKdjU5UEVwYTZIdGtlODh1ZjkwRFBDa2RaV0xnTXRzdnkwREJDclFJREFRQUJBb0lCQUMrZXN0MWEra3VjVTlBcwpkQlE0blZUTFNQaFZxa1NXMjJjbUs4Y3NPVnpCMUJqTU15RVpQQndGOUdxaGt1bVpjZmVpNUJXK0JGd1RpRS9TCjEwSzhDKzRrVmU1OFA3NG81aTQ3Q2d1WkF4eVVhNmkrTDhYN3BITXZQZGFkdEtaRTJ3Y0JHWnRYSkhNZ2V4bHoKaTJpTC9vbXYrZ1ViNkpHdWxqbGhxanExclErUmVBdjJmVFlOR29KbUtHUFhsaW1pOCtLZDhkWHBkS0NEU3RvMgpOUDcrWGNKeE1jZWpWRG1WNFozcncrZW1vUGdyKzlKM0NaMElXVmlEeGlCNFIvMVVPL2l5NmdYU1dhNmxJdlRvCjJKN1Q5bFAzMG4wT3RGTTdTaUw5TEhuYlBPeFNONXd4SkROenc1eEpaWUNMbDQwdU9lcktUVFZXZzBUU0Q2dGEKZE9Zc2VPRUNnWUVBek1hQWZraUp5d1h0ckhnN2ozRUx3WFZQMVNUM1RERGhSa0d2eUtUVExVWUpidFp2bjQ4MQp3MWppajBJK2tEZjdLb0RnVTRDUnN6N0lCdzRwVEEvVWphMFkrV294ODkxTlVzVFg5YktXOHJWZXMxNDc2TzNHCnZzaVpqT1RVMWhHNmlDdCt4ZXV4S1VKQlVnSUFIaVloK0JZQXJtSEwvam5waDZ5UXQ5ZUFQa1VDZ1lFQThBV0sKVGZWeTY0N1Y2Wmw0SGtpYzA0OTA3dzhGb1B1b0pKbkRPaTBPT1FacWdzbHpmbytpb1pNWFpib2hlWVlrMkRWSgpLdXZ3bXlSYnp3clZSMHRsMUtZMUpCUlBPUnZJZ05OcUd4U1VPOVNHdGp5WDJkeGw5NnFyeXpYRWZBZDlPbDZ5CkRVbUhVazNPU0ptUUFVZUlNQWVja3dCbHJQNkFGUHF6YkRseERVa0NnWUFtVzNodUwwQkFXMWZ0Mm9ldGtzdFUKN1ppS29HWUNTNVduQjE3YUFxWGI4U2FMTUhld042OG1iRnZqNGRDK1N2NHRNLzdSSldBbFp6a291U3BiS1F2egpqZDBEdXpTb0xXZnlmUlRHM1RFUlBaQURvVDh2bm5wLytpdHBET1FoUlNqZm9PWWFSTS9tbnBhRnVnSXpvNGp4Clk0MjFISFNOcG1Oc0dwTHhITmpVMFFLQmdRRGNTZWx1dFlnOHc4SjJmR0V2UWl1VlRBUGhhS01jNWZ1alkvQTMKRTZVTEYyVm9GZVpIVVNMTk1rZjJHSWVpbGtoNG15ZEZHbmlnbXhVY0IvQkpWOXprOTRxc1MydU9tWDVESmtZcgpCbzhwL2x1SkpFR21OVTdzcVJxMURlc0Fmb3NqcUVlNFJEVExpQmQ4c0Z2bm9MUlFpSnZlbXkyMlFYcGlSdG1uCnlXeUY2UUtCZ1FDVHNCemRZOVRuWTZXMkgyTnBtVGg0c1VqV3BIRUFXa01hUnJDMkp1b0p3Mkp3TExjRGFFVEMKaGhia1BmY2pnb2draWxoUmRYaktWM29IVkxsN1d6Rlp4OHRqU29LRDU0eVRtV1JoL2JsWDBOZFFmQjhUcnNoWQpONkhqV2RRLys1NFJkOEdlTHRMVkF5b1B1eTRnalNPUnJReWVKWC9US01CV1hqcVVyL0pyN1E9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
is this a real key? remove it if so

Reply:
it's not,
took it from e2e share 4 just for not having execption